src/register/ThirdPartyAuth.jsx

```diff
@@ -24,6 +24,7 @@ const ThirdPartyAuth = (props) => {
   const isInstitutionAuthActive = !!secondaryProviders.length && !currentProvider;
   const isSocialAuthActive = !!providers.length && !currentProvider;
   const isEnterpriseLoginDisabled = getConfig().DISABLE_ENTERPRISE_LOGIN;
+  const ThirdPartyAuthPreloader = isSocialAuthActive || (isEnterpriseLoginDisabled && isInstitutionAuthActive);
 
   return (
     <>
```

Comment:
The comment regarding using the same variable for the heading condition also applies here.

Reply:
I have changes all `ThirdPartyAuthPreloader` to `isThirdPartyAuthActive`